strings/strings.c

```diff
@@ -10,6 +10,14 @@
 int string_length(char *s)
 {
 
+    int c = 0;
+
+    while (*s != '\0')
+    {
+        c++;
```

Comment:
Good job on your work Tyrone! & I don't think you needed to increment or return c here